js/renovation/ui/scheduler/workspaces/base/time_panel/layout.tsx

```diff
@@ -39,6 +32,8 @@ export const viewFunction = (viewModel: TimePanelTableLayout) => (
 @ComponentBindings()
 export class TimePanelTableLayoutProps extends LayoutProps {
   @OneWay() className?: string;
+
+  @OneWay() viewData?: GroupedViewData;
```

Comment:
This prop is already present in LayoutProps. It is better to remove it from here